frontend/src/__mocks__/mockStorageClasses.ts

```diff
@@ -45,8 +38,10 @@ export const buildMockStorageClass = (
   },
 });
 
-export const mockStorageClasses: MockStorageClass[] = [
+export const mockStorageClasses: StorageClassKind[] = [
   {
+    apiVersion: 'storage.k8s.io/v1',
+    kind: 'StorageClass',
```

Comment:
FWIW, `apiVersion` & `kind` are not returned within individual storage class objects, which is why those were omitted originally. 

This is the format that actually gets returned from the GET for storage classes:
```
{
    "kind": "StorageClassList",
    "apiVersion": "storage.k8s.io/v1",
    "items": [
        {
            "metadata": {
                ...
            }
        }
    ]
}
```

Reply:
Yeah thats true, but it makes the types work with `cy.interceptK8sList` so you can do things like this
```tsx
cy.interceptK8sList(PVCModel, mockK8sResourceList([mockPVCK8sResource({})]));
```
it also just follows the rest of how our mocks work together